### PR TITLE
Fix unit tests and handle SF Form BC introduced in 2.7

### PR DESCRIPTION
**Pull request by Nakard**
- Add appropriate PHPDocs for inspections
- PSR-2 fixes in few places
- Put long texts/code used in assertions  into separate files
- Change fixtures related to DunglasApiBundle to reflect
  changes in mentioned bundle introduced with new mapping
  loaders
- Skip caching extractor test if SYMFONY_VERSION is 2.4.*

Related to issue #699

### CODE REVIEW

File: Tests/Extractor/CachingApiDocExtractorTest.php [hunk 1]
```diff
@@ -64,7 +70,7 @@ public function testCachedResultSameAsGenerated($view)
 
         $cacheFile = $container->getParameter('kernel.cache_dir').'/api-doc.cache';
 
-        $expectedViewCacheFile = $cacheFile.'.'.$view;
+        $expectedViewCacheFile = $cacheFile . '.' . $view;
```
Comment (oktapodia) [inline Tests/Extractor/CachingApiDocExtractorTest.php]: Add a single space around binary operators (==, &&, ...), with the exception of the concatenation (.) operator;

**Comment by oktapodia** [comment]
:+1:

**Comment by Nakard** [comment]
@oktapodia Spaces around concat operator removed

**Comment by oktapodia** [comment]
I'm not in the Nelmio team sorry

ping @willdurand

**Comment by AbdoulNdiaye** [comment]
@Nakard Maybe I'm wrong but I think your work has been merged here: https://github.com/nelmio/NelmioApiDocBundle/commit/e66fb209b56e7937647fae39e64b6218efa62e1e
But the author has changed :laughing:

**Pull request closed without merging**